util/fips-checksums.sh

```diff
@@ -8,7 +8,7 @@ for f in "$@"; do
     # the '*stdin' marker to the filename preceded by a space.  See the
     # sha1sum manual for a specification of the format.
     case "$f" in
-        *.c | *.h )
+        *.c | *.h | *.h.in | *.inc)
```

Comment:
I'm not certain what's best here.  @levitte suggestions?
There are `.c.in` files which are currently not cleansed of ifdefs

Reply:
Those .c.in should definitely be checksummed as well - this is #15192.

Reply:
Should they be run through cpp though?

Reply:
You mean run through the lang-compress.pl as 'C'  and unifdef? IMO it should work for those fine.

Reply:
I've added `*.c.in` to this case as well.